lib/ReactViews/Loader.tsx

```diff
@@ -1,9 +1,8 @@
 import { TFunction } from "i18next";
```

Comment:
Also might be better to move this to `lib/Styled`

Reply:
I'm a bit confused about that. How do we now decide what goes in `lib/Styled`?

Reply:
Not sure, but I see `Loader` as a component that is reused in combination with other components and doesn't stand alone. It just might be easier to find it later when needed.